bitnami/discourse/templates/deployment.yaml

```diff
@@ -67,10 +76,11 @@ spec:
             - sh
             - -c
             - |
-              mkdir -p "/bitnami/discourse"
-              chown -R "discourse:root" "/bitnami/discourse"
-          securityContext:
-            runAsUser: 0
+              mkdir -p /bitnami/discourse
+              find /bitnami/discourse -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R {{ .Values.discourse.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}
```

Comment:
just fyi,
i get this error in my init container, which indicates that no files were found

```bash
chown: missing operand after '0:0'
```

in another chart https://github.com/bitnami/charts/issues/3863 they had similar issues and fixed it with `-mindepth 0` as in include the parent directory in case there are no files